### PR TITLE
Fix Python search

### PR DESCRIPTION
**Pull request by jakirkham**
This seemed to work ok for me locally. It seemed to require using the non-versioned environment variables to work. Also, it seems to its own thing with site-packages anyways, but this is at least an attempt to get that right too.

### CODE REVIEW

**Comment by jakirkham** [comment]
So AppVeyor is off to the races again. Sorry. :facepalm:

File: recipe/build.sh [hunk 1]
```diff
@@ -11,11 +11,15 @@ if [ "${SHORT_OS_STR}" == "Darwin" ]; then
     OPENMP=""
 fi
 
+INC_PYTHON="${PREFIX}/include/python${PY_VER}"
+SPD_PYTHON="${PREFIX}/lib/python${PY_VER}/site-packages"
```
Comment (patricksnape) [inline recipe/build.sh]: Doesn't this exist already as `SP_DIR`?

Comment (jakirkham) [inline recipe/build.sh]: Good point.

Comment (jakirkham) [inline recipe/build.sh]: Fixed.

**Pull request merged**